src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -47,12 +47,12 @@ Notes on NerdGraph requirements:
 * NerdGraph requires a New Relic [user key](/docs/apis/intro-apis/new-relic-api-keys/#user-api-key): this can be generated and accessed from the [GraphiQL explorer](#explorer).
 * User type and assigned roles can affect your NerdGraph permissions. For more details, see [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data#user-permissions).
 
-The NerdGraph endpoints are:
+The endpoint depends on your [data center region](/docs/accounts/accounts-billing/account-setup/choose-your-data-center):
 
-* Main endpoint: `https://api.newrelic.com/graphql`
-* Endpoint for accounts using EU data center: `https://api.eu.newrelic.com/graphql`
+* US data center region: `https://api.newrelic.com/graphql`
+* EU data center region: `https://api.eu.newrelic.com/graphql`
 
-To access the endpoint, use the following cURL command:
+To access the endpoint, you can [use our NerdGraph explorer](#explorer), or you can access it directly with a cURL command like:
```

Comment:
```suggestion
To access the endpoint, you can use our [NerdGraph explorer](#explorer), or you can access it directly with a cURL command like:
```

Reply:
@a-sassman thanks for the tip, I used this with a minor modification: To access the endpoint, you can use our [NerdGraph explorer](#explorer), or you can access it directly with a curl command similar to this: